Rakefile

```diff
@@ -91,4 +91,4 @@ task :convert do
   result = `bin/kramdown --input=html --output=kramdown`
   puts 'Converted text:'.magenta
   puts result.bold
-end
+end
```

Comment:
Whatsnew updated PR must not change any other files but `src/_data/whats-new.yml`, especially when this is an infrastructure file. I don't believe that an author and a reviewer didn't notice this change.